src/components/Match/matchColumns.jsx

```diff
@@ -438,18 +431,18 @@ export const supportColumns = [
 export const chatColumns = [
   {
     displayName: strings.filter_is_radiant,
-    field: 'team',
-    displayFn: row =>
+    displayFn: row => 
       <div className={styles.chatTeamIcon}>
         {
           row.isRadiant ?
-            <IconRadiant className={styles.iconRadiant} /> :
-            <IconDire className={styles.iconDire} />
+          <IconRadiant className={styles.iconRadiant} /> :
+          <IconDire className={styles.iconDire} />
         }
       </div>
     ,
   },
-  heroTdChatColumn, {
+  Object.assign(heroTdColumn, {sortFn: false}),
```

Comment:
you need to make {} the first argument.  Or just use the rest syntax.  I pointed out it's not officially part of ES6 yet, but we are using it all over the place anyway, and babel transpiles it for us.